From: Carlos Maddela <e7appew@gmail.com>
Date: Thu, 5 Jan 2017 19:14:35 +1100
Subject: Apply pdftex workarounds for build reproducibility.

Description: Apply pdftex workarounds for build reproducibility.
 * Generate the same PDF IDs.
 * Suppress additional metadata for included images.
Author: Carlos Maddela <e7appew@gmail.com>
Forwarded: not-needed
Last-Update: 2017-08-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 documentation/user-manual/manual.tex | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/documentation/user-manual/manual.tex b/documentation/user-manual/manual.tex
index 60fb51e..bf4364d 100644
--- a/documentation/user-manual/manual.tex
+++ b/documentation/user-manual/manual.tex
@@ -65,6 +65,8 @@
 \fancyfoot{}
 \fancyfoot[LE,RO]{page \thepage\ of \pageref{LastPage}}
 
+\pdftrailerid{dvdisaster}
+\pdfsuppressptexinfo=-1
 \begin{document}
 
 \definecolor{lightorange}{RGB}{255,224,150}
